nltk/translate/util.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+
+def jacknifing(score_list, averaging=True):
+        ''' This is a averaging function for calculating ROUGE score
+    when multiple references are present. This technique has been
+    referred to as the Jacknifing technique in the original paper.
+    score_list = list of scores over which averaging occurs
+    '''
+        if(len(score_list) == 1):
+            return np.mean(score_list)
```

Comment:
Is numpy usage here really necessary? Python 3 has `statistics` as native and if otherwise, it's not hard to do `sum(score_list)/len(score_list)`.

Reply:
Done